### ISSUE TITLE
javacv reports PICVideo 4 license requirement

### ISSUE THREAD

**Issue by radu-ta** [bytedeco/javacv#1380]
Hello,
I am using javacv for a project, the maven depex are below:
<dependency>
			<groupId>org.bytedeco</groupId>
			<artifactId>javacv-platform</artifactId>
			<version>1.5.2</version>
		</dependency>
		<dependency>
    		<groupId>org.bytedeco</groupId>
    		<artifactId>openblas-platform</artifactId>
    		<version>0.3.7-1.5.2</version>
		</dependency>
		<dependency>
    		<groupId>org.bytedeco</groupId>
    		<artifactId>opencv-platform</artifactId>
    		<version>4.1.2-1.5.2</version>
		</dependency>
After using:
OpenCVFrameGrabber grabber = new OpenCVFrameGrabber(deviceWebcamId); 
grabber.setImageWidth(imageWidth);
grabber.setImageHeight(imageHeight);

and setting a resolution higher than: 
1600 x 1200 e.g.: 3264x 2448 and 2592 x 1944

I get the message attached below:
![grafik](https://user-images.githubusercontent.com/61418735/75157059-ac018f80-5713-11ea-8898-855503bf42b2.png)

The message sais that I have to buy a license for the product.
Is there any workaround for this?

**Comment by saudet** [comment]
That's something specific to your machine, not JavaCV. Try to reinstall the drivers of your camera.

**Comment by radu-ta** [comment]
Hello Samuel,

Thanks for answering.
At the beginning I was also thinking about the drivers. But I used the app with which the [tablet](https://www.getac.com/wp-content/uploads/2018/12/Getac_F110_UK_20190820.pdf) comes equipped and the issue does not pop up using the 3264x 2448 and 2592 x 1944 resolutions. So I used the procexp64.exe from SysInternals, the "Find Window's Process" functionality, and the result is:
![20200227_094536](https://user-images.githubusercontent.com/61418735/75427624-249c6200-5947-11ea-985b-821690478857.jpg)

From my findings it still points to JavaCV.

**Comment by saudet** [comment]
You should try another pixel format. The recommended one should be in the
documentation of the camera.

**Comment by saudet** [comment]
Also, OpenCV doesn't usually work very well for video capture.
Try to use `VideoInputFrameGrabber` instead.

**Comment by radu-ta** [comment]
Hello Saudet,

Thank you for your help and guidance.
I ended using the OpenCVFrameGrabber, but that did not solve the resolution problem.
After some digging into the camera's drivers, I found that PicVideo 4 is indeed embedded in the drivers. My bad for blaming JavaCV. 
I contacted the manufacturer of the tablet by phone and by email but no response came.
In the end I used the lower resolution for finishing the project.
